### DISCUSSION TITLE
Nokee 0.4 Javadoc

### DISCUSSION THREAD

**Discussion by pedrolamarao** [nokeedev/gradle-native#573] [Q&A] [answered]
Hey @lacasseio thanks once more for your great work!
The Javadoc links in nokee.dev have been updated to version 0.5.
I am assuming from development history that 0.5 is very different than 0.4.
Are there Javadocs for 0.4 available online somewhere?
Or is it perhaps time to start testing 0.5?

**Comment by lacasseio** [comment, marked as answer]
I understand the confusion, that change happen a while back when I nudged all the documentation toward a live-at-head philosophy. Unfortunately, at the time, the documentation was versioned using the next version + commit hash. The version has since been reverted back to 0.4.x but the documentation pipeline had some little issues which prevented from releasing a proper version. There haven't been too many meaningful changes to the documentation so it's a bit lower in the priority.

For the next release, which I agree is dragging quite a bit, I'm currently cleaning up the documentation pipeline to something more stable (I still need to rehook the DSL generation and aggregated Javadoc). The last part that I'm also starting soon is official performance tests to hunt down any performance regression.

Once those are done, we will be in a good position to release versions 0.5 to 0.9 almost back-to-back. For now, I would say there is no action for you.
